### PR TITLE
Try a native aarch64 build

### PR DESCRIPTION
**Pull request by Cadair**
for science

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ jobs:
 
   test:
     needs: [core, sdist_verify, rust]
-    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@main
+    uses: sunpy/github-actions-workflows/.github/workflows/tox.yml@main
```
Comment (Cadair) [inline .github/workflows/ci.yml:64]: Apparently GH wont let you run cross-organization reuseable workflows on self-hosted or paid-for runners.

**Review by dpshelio — APPROVED** [review]

**Review by nabobalis — APPROVED** [review]

**Comment by Cadair** [comment]
Before this gets there's a few things that need sorting, 1) a fork of the workflows (apparently) 2) the publish workflow needs to add aarch64 support 3) I need to be ok with paying for this.

**Comment by Cadair** [comment]
I have made it so the sunpy fork of the workflows pulls weekly on a cron, so it shouldn't drift out of date. I now just need to make the publish workflows work.

**Comment by Cadair** [comment]
This works when using https://github.com/OpenAstronomy/github-actions-workflows/pull/247, the only real downside is the need to use the workflows from within the org.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -118,7 +120,7 @@ jobs:
         contains(github.event.pull_request.labels.*.name, 'Run publish')
       )
     needs: [test, docs]
-    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish.yml@main
+    uses: sunpy/github-actions-workflows/.github/workflows/publish.yml@publish-runs-on
```
Comment (Cadair) [inline .github/workflows/ci.yml]:
```suggestion
    uses: sunpy/github-actions-workflows/.github/workflows/publish.yml@main
```

**Pull request merged**